website/client/src/components/snackbars/notifications.vue

```diff
@@ -19,40 +29,294 @@
     width: 350px;
     z-index: 1400; // 1400 is above modal backgrounds
 
-    &-top-pos {
-      &-normal {
-        top: 65px;
-      }
+    top: var(--current-scrollY);
 
-      &-sleeping {
-        top: 105px;
-      }
-    }
+    justify-content: flex-end;
+    display: flex;
+  }
+
+  .animations-holder {
+    position: relative;
+    display: block;
+  }
+
+  .notification-item {
+    transition: transform 0.25s ease-in, opacity 0.25s ease-in;
+  }
+
+  .notifications-move {
+    // transition: transform .5s;
+  }
+
+  .notifications-enter-active {
+    // transition: opacity .5s;
+  }
+
+  .notifications-leave-active {
+    position: absolute;
+    right: 0;
+  }
+
+  .notifications-enter,
+  .notifications-leave-to {
+    opacity: 0;
+    right: 0;
   }
 </style>
 
 <script>
 import { mapState } from '@/libs/store';
 import notification from './notification';
+import { sleepAsync } from '../../../../common/script/libs/sleepAsync';
+
+const notificationsVisibleAtOnce = 4;
+const removalInterval = 2500;
+const delayDeleteAndNew = 60;
+const delayFilling = 240;
```

Comment:
For magic numbers and other simple constants defined at top of file, we usually use ALL_CAPS_SNAKE naming convention

Reply:
will change those 👍